engine/orchestration/src/main/java/com/cloud/vm/VirtualMachineManagerImpl.java

```diff
@@ -881,8 +881,9 @@ protected boolean checkWorkItems(final VMInstanceVO vm, final State state) throw
             }
 
             if (state != State.Stopped) {
-                s_logger.debug("VM " + vm + " is not in a state to be started: " + state);
-                return null;
+                String msg = String.format("Cannot start VM: %s in %s state", vm, state);
```

Comment:
`vm` is of type `VMInstanceVO`; This class implements a `toString()` like `VM instance...`, so the final message will be:

`Cannot start VM: VM instance...`.

We could remove the `VM:` here.

Reply:
keen observation